packages/tsconfig.declarations.json

```diff
@@ -0,0 +1,6 @@
+{

```

Comment:
I think it's better to keep all the tsconfigs at the same level. Is there a reason why it's here? Would that be possible to move it inside the root folder?